tools/view_model.cpp

```diff
@@ -311,14 +311,14 @@ int main(int argc, char **argv)
     info_msg << std::endl;
     info_msg << "Keys:" << std::endl;
     info_msg << "    r - reload model" << std::endl;
-    info_msg << "    v - show / hide model volumes" << std::endl;
+    info_msg << "    v - hide all volumes, show model volumes, show room volumes" << std::endl;
     info_msg << "    c - circle rotate" << std::endl;
     info_msg << "    p - snap pitch" << std::endl;
     info_msg << "    q - quit" << std::endl;
 
     std::cout << info_msg.str();
 
-    bool show_volumes = true;
+    int show_volumes = 1; // 0= no volumes, 1=entity volumes, 2=room volumes
```

Comment:
This will be confusing. I'd make the values to check against enums or at least CONSTANTS